src/main/java/com/treasure/hunt/strategy/searcher/impl/StrategyFromPaper.java

```diff
@@ -232,38 +232,39 @@ private void setRectToPhase() {
         D = JTSUtils.createPoint(startX - halfDiff, startY - halfDiff);
     }
 
-    private Movement rectangleScan(Point A, Point B, Point C, Point D) {
+    public static Movement rectangleScan(Point A, Point B, Point C, Point D) {
         Movement movements = new Movement();
 
         int k = (int) A.distance(B);
```

Comment:
```suggestion
        int roundedDistance = (int) A.distance(B);
```
Idk something more descriptive. Refactoring the code (or even just renaming vars) would make it more readable and accelerate the merge and review times a lot.